tema2.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <title>OFF-WHITE</title>
+    <head>
+        <body>
+            <fieldset>
+                <legend>OFF-WHITE</legend>
+                <label>
+                    <p>Your name
+                        <input required type="text" name="your-name" placeholder="write here..." min="3" max="10"/>
+                    </p>
+                    <p>Your email
+                        <input required type="text" name="your-email" placeholder="write here..."/>
+                    </p>
+                    <p>Stock:
+                        <input type="radio" name="hoodies" value="hoodies"/>
+                        <span>Hoodies</span>
+                        <input type="radio" name="pants" value="pants"/>
+                        <span>Pants</span>
+                        <input type="radio" name="jackets" value="jackets"/>
+                        <span>Jackets</span>
+                    </p>
+                    <p>Colours:
+                        <select name="optiuni">
+                            <option value="black">Black</option>
+                            <option value="white">White</option>
+                            <option value="red">Red</option>
+                            <option value="blue">Blue</option>
+                            <option value="green">Green</option>
+                        </select>
+                    <p>Numbers:
+                        <input type="number" name="number" value="1" min="1" max="7"/>
+                    </p>
+                    </p>
+                    <p>Size:
+                        <input type="checkbox" name="s" value="s"/>
+                        <span>S</span>
+                        <input type="checkbox" name="m" value="m"/>
+                        <span>M</span>               
+                        <input type="checkbox" name="l" value="l"/>
+                        <span>L</span>
+                        <input type="checkbox" name="xl" value="xl"/>
+                        <span>XL</span>
+                    </p>
+                    <textarea required name="write-here" rows="10" cols="60">Write here...
+                    </textarea>
+                    <p>Delivery date:
+                        <input type="date" name="data-livrare" max="2023-12-24"/>
+                    </p>
+                    <form action="https://www.off---white.com/en-ro/?gclid=EAIaIQobChMIqOffqPmr_wIV94toCR37TAmVEAAYASAAEgIHkvD_BwE&gclsrc=aw.ds" method="get" target="_blank">
+                        <input type="submit" value="Submit">
+                        <input type="reset" value="Reset">
```

Comment:
Toate elementele de mai sus trebuie sa fie in `<form>` 
aici `<form>` nu are tagul de inchidere `</form>`